code/API_definitions/qod-api.yaml

```diff
@@ -222,7 +222,38 @@ paths:
         - oAuth2ClientCredentials: []
         - threeLegged:
             - "qod-sessions-write"
-
+    get:
+      tags:
+        - QoS sessions
+      summary: Get all active sessions information
+      description: Get information about all active sessions authorized to be retrieved by the provided access token.
+      operationId: getSessions
+      parameters:
+      - name: phoneNumber
```

Comment:
It would be better to filter on $ref: "#/components/schemas/Device and not just phone number.